.gitlab/package_build/apk.yml

```diff
@@ -13,26 +17,15 @@ agent_android_apk:
       when: never
     - when: on_success
   stage: package_build
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/android_builder:$DATADOG_AGENT_BUILDERS
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/android_builder:v3715617-76ed3eb
```

Comment:
Updates the Android NDK. We can update it once the PR on the builders repo gets merged.